src/Compute/Compute/help/Add-AzVmssSecret.md

```diff
@@ -27,8 +27,32 @@ For more information about the cmdlets, see [Azure Key Vault Cmdlets](/powershel
 
 ## EXAMPLES
 
-### Example 1: Add a secret to the VMSS
+### Example 1: Add a secret to the VMSS using the Azure Key Vault virtual machine extension
+
+```powershell
+# Build settings
+PS C:\> $settings = @{
+    secretsManagementSettings = @{
+        pollingIntervalInS       = "<pollingInterval>"
+        certificateStoreName     = "<certStoreName>"
+        certificateStoreLocation = "<certStoreLoc>"
+        observedCertificates     = @("<observedCert1>", "<observedCert2>")
+    } 
+} | ConvertTo-Json
+PS C:\> $extName = "KeyVaultForLinux"
+PS C:\> $extPublisher = "Microsoft.Azure.KeyVault"
+PS C:\> $extType = "KeyVaultForLinux"
+# Add Extension to VMSS
+PS C:\> $vmss = Get-AzVmss -ResourceGroupName <ResourceGroupName> -VMScaleSetName <VmssName>
+PS C:\> Add-AzVmssExtension -VirtualMachineScaleSet $vmss  -Name $extName -Publisher $extPublisher -Type $extType -TypeHandlerVersion "2.0" -Setting $settings
+# Start the deployment
+PS C:\> Update-AzVmss -ResourceGroupName <ResourceGroupName> -VMScaleSetName <VmssName> -VirtualMachineScaleSet $vmss
 ```
+
+To install certificates on a virtual machine it is recommended to use the [Azure Key Vault virtual machine extension for Linux](https://docs.microsoft.com/azure/virtual-machines/extensions/key-vault-linux) or the [Azure Key Vault virtual machine extension for Windows](https://docs.microsoft.com/azure/virtual-machines/extensions/key-vault-windows). 
```

Comment:
Shouldn't the above statement be added before the Example 1 that provides the steps to add a secret to the VMSS using the Azure Key Vault extension, also we should mention virtual machine scaleset vm or someting similar that points to the scaleset resource which is different than the stand alone vm resource.

Reply:
I basically just inserted the extension info into what was already there. I will revisit this and see how to make the order more clear.